### PR TITLE
creating css and html for contact page

### PR DESCRIPTION
**Pull request by Pierre-Remy**
I am starting to work on the contact page.

### CODE REVIEW

File: landing.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            <p>pierreremynshimiyimana143@gmail.com</p>
+            <img src="D:\Andela Training\Week 2\Personal website" class="emai" alt="email icon"/>
+        </div>
+        <div id="tel">
+            <hr>(+250) 783324773 <hr>
+            <img src="D:\Andela Training\Week 2\Personal website" class="te" alt="telephone icon" />
+        </div>
+        <div id="linkedlin">
+                <img src="https://www.google.com/search?q=linkedin+icon&sxsrf=ALeKk02Z0T8WRHdZE7CrI7yYNn8AaHd2zA:1595251526447&tbm=isch&source=iu&ictx=1&fir=JdyXos__tp7KHM%252CvLBuaTBzrfpOUM%252C_&vet=1&usg=AI4_-kQskxX0Qym2UBqOZtwEovl0j_1ryQ&sa=X&ved=2ahUKEwjCpr6E99vqAhVPUBUIHQ4GAxcQ9QEwAHoECAoQKQ&cshid=1595251563373225#imgrc=JdyXos__tp7KHM" alt="linkedin icon" />
+        </div>
```
Comment (Niyiragira) [inline landing.html:45]: Keep up bro this is amazing

Comment (Pierre-Remy) [reply landing.html:45]: Thank you for the feedback

**Comment by Pierre-Remy** [comment]
Received feedback from Gerard.

**Review by Pierre-Remy — COMMENTED** [review]
The CSS and HTML are well done.

**Pull request merged**